modules/t/test-genome-DBs/circ/core/xref.txt

```diff
@@ -128,13 +128,13 @@
 88581	50728	CP003765.1:CDS:complement(234076..235323)	CP003765.1:CDS:complement(234076..235323)	0	\N	NONE	
 88582	50727	CP003765.1:CDS:complement(234076..235323)	CP003765.1:CDS:complement(234076..235323)	0	\N	NONE	
 88583	50719	BTF1_31777 1-1	BTF1_31777-1	0	\N	NONE	
-88584	50726	CP003765.1:CDS:complement(join(235337..235425,1..685))	CP003765.1:CDS:complement(join(235337..235425,1..685))	0	\N	NONE	
+88584	50726	CP003765.1:CDS:complement(join(235337..235425,1..6	CP003765.1:CDS:complement(join(235337..235425,1..685))	0	\N	NONE	
```

Comment:
funny, looks like our test data had some accessions which were already way too long for the table definition